corehq/apps/export/views.py

```diff
@@ -404,37 +403,6 @@ def _get_download_task(self, in_data):
             filename=self._get_filename(export_instances)
         )
 
-    def _get_and_rebuild_export_schema(self, export_id):
-        # TODO
-        raise Exception("this must be failing if it's ever called...")
-
-    def _get_bulk_download_task(self, export_specs, export_filter):
```

Comment:
References to this method and the one below it were both removed when I overwrote `_get_download_task` in the previous commit.